### DISCUSSION TITLE
Rename span_id to spanId and trace_id to traceId

### DISCUSSION THREAD

**Discussion by rakesh-eltropy** [open-telemetry/opentelemetry-java#5528] [Q&A]
Hi All, Otel Java agent is automatically providing trace_id and span_id.

`DEBUG org.springframework.web.servlet.DispatcherServlet:1135 - Completed 200 OK {span_id=c1fa3ed7625b1162, trace_flags=01, trace_id=702867124dd946a9a91d7abda6c3ca6d}`

Is there a way to rename span_id to spanId and trace_id to traceId?

We want to do this to make it consistent with our golang apps traces..

**Comment by trask** [comment]
the names span_id and trace_id are specified here: https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/compatibility/logging_trace_context.md#overview

though there's an open issue for renaming these to spanId and traceId: https://github.com/open-telemetry/opentelemetry-specification/issues/3518

**Reply by jack-berg** [reply]
Any idea why golang isn't aligned with the specification?
